### PR TITLE
ISID and MAC Filter Creation

### PR DESCRIPTION
**Pull request by sbarguil**
Innitial Commit #4

### CODE REVIEW

**Comment by sbarguil** [comment]
@sbarguil To include Features in the YANG for bth filtering options.

File: yang/ietf-acl-enh.yang [hunk 1]
```diff
@@ -62,6 +62,16 @@ module ietf-acl-enh {
       "Match based on a pattern is supported.";
   }
 
+  feature match-on-vlan-flter {
```
Comment (boucadair) [inline yang/ietf-acl-enh.yang:65]:
```suggestion
  feature match-on-vlan-filter {

```

File: yang/ietf-acl-enh.yang [hunk 2]
```diff
@@ -274,6 +284,111 @@ module ietf-acl-enh {
     }
   }
 
+  augment "/acl:acls/acl:acl/acl:aces"
+        + "/acl:ace/acl:matches/acl:l2" {
+    description
+      "Handle the augmentation of MAC VLAN Filter.";
+    container vlan-filter {
+      if-feature "match-on-vlan-flter";
```
Comment (boucadair) [inline yang/ietf-acl-enh.yang:292]:
```suggestion
      if-feature "match-on-vlan-filter";

```

File: yang/ietf-acl-enh.yang [hunk 3]
```diff
@@ -274,6 +284,111 @@ module ietf-acl-enh {
...
   }
 
+  augment "/acl:acls/acl:acl/acl:aces"
+        + "/acl:ace/acl:matches/acl:l2" {
+    description
+      "Handle the augmentation of MAC VLAN Filter.";
+    container vlan-filter {
+      if-feature "match-on-vlan-flter";
+      description
+        "Indicates how to handle MAC VLAN.";
```
Comment (boucadair) [inline yang/ietf-acl-enh.yang:294]:
```suggestion
        "Indicates how to handle MAC VLANs.";

```

File: draft-ietf-netmod-acl-extensions.md [hunk 4]
```diff
@@ -585,6 +585,93 @@ In order to support rate-limiting (see {{ps-rate}}), a new action called "rate-l
 ~~~
 {: #example_5 title="Example Rate-Limit Incoming TCP SYNs"}
 
+## ISID Filter
+
+Provider backbone bridging (PBB) was originally defined as IEEE
+802.1ah standard. However, instead of multiplexing VLANs, PBB
```
Comment (boucadair) [inline draft-ietf-netmod-acl-extensions.md:591]: Can we add a reference? Thanks.

File: draft-ietf-netmod-acl-extensions.md [hunk 5]
```diff
@@ -585,6 +585,93 @@ In order to support rate-limiting (see {{ps-rate}}), a new action called "rate-l
...
+the provider domain, by encapsulating it in a 24 bit instance service
+identifier (I-SID). This allows for complete transparency between the
+customer network and the provider network.
+
+The I-component forms the customer or access facing interface or
+routing instance. The I-component is responsible for mapping customer
+Ethernet traffic to the appropriate I-SID. In the netrowk is
+mandatory to configure the default service identifier.
+
+Beign able to filter by I-Component Service identifier is a feature of
```
Comment (boucadair) [inline draft-ietf-netmod-acl-extensions.md:602]:
```suggestion
Being able to filter by I-component Service identifier is a feature of
```

File: draft-ietf-netmod-acl-extensions.md [hunk 6]
```diff
@@ -585,6 +585,93 @@ In order to support rate-limiting (see {{ps-rate}}), a new action called "rate-l
...
+      }
+    }
+   }
+~~~
+{: #example_6 title="Example ISID Filter"}
+
+## VLAN Filter
+
+The flexibility to apply a traffic filter to a VLAN is part of the VPN control
+requirements. So, all packets that are bridged within a VLAN or that
```
Comment (boucadair) [inline draft-ietf-netmod-acl-extensions.md:641]: Any pointer to cite here?

**Review by boucadair — CHANGES_REQUESTED** [review]

**Pull request merged**